src/components/organisms/FilterBar.tsx

```diff
@@ -62,9 +62,15 @@ const FilterBar: FC<IProps> = () => {
                   </Button.Standard>
                 </Grid>
               </Grid>
-              <Grid item>
-                <Box mt={1}>{isDescOpen && <Typography.Body1>{store.activeNewsFlash?.title}</Typography.Body1>}</Box>
-              </Grid>
+              {isDescOpen && (
+                <Grid item>
+                  <Box mt={1}>
+                    <Typography.Body1>
+                      {store.activeNewsFlash?.description || store.activeNewsFlash?.title}
```

Comment:
To my understanding, the `description` field contains the `title` so I used `title` as a fallback over displaying both of them.